File: Java/Program/Penyimpanan.java
public class Penyimpanan extends Komponen {
    private int kapasitasGB;
    private String tipe;

    public Penyimpanan(String merk, String nama, int kapasitasGB, String tipe) {
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.tipe = tipe;
    }

    public void setKapasitasGB(int kapasitasGB) {
        this.kapasitasGB = kapasitasGB;
    }

    public int getKapasitasGB() {
        return kapasitasGB;
    }

    public void setTipe(String tipe) {
        this.tipe = tipe;
    }

    public String getTipe() {
        return tipe;
    }
}

File: Python/Program/Penyimpanan.py
from Komponen import Komponen

class Penyimpanan(Komponen):
    def __init__(self, merk, nama, kapasitasGB, tipe):
        super().__init__(merk, nama)
        self.kapasitasGB = kapasitasGB
        self.tipe = tipe
        
    def set_kapasitasGB(self, kapasitasGB):
        self._kapasitasGB = kapasitasGB
        
    def get_kapasitasGB(self):
        return self._kapasitasGB
    
    def set_tipe(self, tipe):
        self._tipe = tipe
        
    def get_tipe(self):
        return self._tipe